pliers/support/setup_yamnet.py

```diff
@@ -0,0 +1,41 @@
+import os
+from io import BytesIO
+from zipfile import ZipFile
+from urllib import request
+from pathlib import Path
+import sys
+import runpy
+
+DOWNLOAD_PATH = Path.home() / 'pliers_data'
+YAMNET_PATH = DOWNLOAD_PATH / 'models-master' / 'research' / 'audioset' / 'yamnet'
```

Comment:
I assume this mirrors the structure in the yamnet repo, but it's a lot of nesting. Unless the yamnet code requires that structure, I'd probably just store this under `.pliers/models/yamnet` or something like that (note that I believe we already use the convention of storing data in `.pliers` in other places, which is what many other packages do as well).

Reply:
I think it's that the whole TF repo is being downloaded. That said, in the download script you could download a subset, or download to a tmp dir, and only copy over what you need to `.pliers`

Reply:
It is now so that the download script fetches the TF repo's zip archive (no way, as far as I can see, to only download a subset), extracts it all into a `yamnet_temp` folder in `pliers_data`, moves the yamnet part to a flat directory `pliers_data/yamnet` and deletes `yamnet_temp`.
We could in principle avoid extracting the whole thing, and just looping through files in the zip archive so to only extract the yamnet ones. That would however add a bunch of lines of code, as we would: 1) have to loop; 2) take all the extracted files and move them to a flat folder, as zipfile keeps the file structure when using `extract`. And after all, the whole thing is only 125Mb.
Just to make sure though, you are talking about `pliers_data` (same default folder in user's home directory as text dictionaries are dumped to), right? Not `.pliers`...

Reply:
Oh, I was under the impression we're putting stuff in `.pliers`, but if we're currently putting it in `pliers_data`, then yeah, putting this there as well makes sense.